### PR TITLE
SOMA refactoring

### PR DESCRIPTION
**Pull request by daniel86**
As discussed last week, this pull request addresses unifying the IRI namespace over all OWL files.
This includes:
- rename OWL files from "EASE*" to "SOMA*"
- xml base for every OWL file is "http://www.ease-crc.org/ont/SOMA.owl"
- all SOMA entities are defined in the namespace "http://www.ease-crc.org/ont/SOMA.owl#"
- additionally, deleted all "package://" imports and replaced them with proper http imports

### CODE REVIEW

**Comment by daniel86** [comment]
probably not really possible to review, and it will cause conflicts with the other pending pull request. But we have discussed to do the refactoring first, before anything else. And it might take some more time to get the SOMA-SAY pull request through

**Comment by mpomarlan** [comment]
If I remember, no, we said we merge the SAY pull request and then this one, precisely to avoid breaking git's ability to merge.

**Comment by daniel86** [comment]
> If I remember, no, we said we merge the SAY pull request and then this one, precisely to avoid breaking git's ability to merge.

nope, definitely not. We said we merge the design request, and then we do the refactoring.

**Comment by mpomarlan** [comment]
That's not how I remember it. The SOMA-SAY was opened 13 days ago when the Design PR was merged, because we all knew it was in the pipeline and it would make sense to get it in before the big refactoring.

**Comment by daniel86** [comment]
> That's not how I remember it. The SOMA-SAY was opened 13 days ago when the Design PR was merged, because we all knew it was in the pipeline and it would make sense to get it in before the big refactoring.

well then you remember it wrong.
We discussed in the meeting before I merged the then opened design request to only merge this one, then do the refactoring. I am 100% sure.

**Comment by mpomarlan** [comment]
I don't know why we would have agreed to such a thing, when we knew SOMA-SAY had a PR in the pipeline and that merging it will be more difficult after the refactoring.

**Comment by daniel86** [comment]
> I don't know why we would have agreed to such a thing, when we knew SOMA-SAY had a PR in the pipeline and that merging it will be more difficult after the refactoring.

yeah well we cannot change that. So let's wait until everyone is back.
I'll be not available for two weeks now.

**Comment by mpomarlan** [comment]
The idea was to get the SAY PR in while Vanja was still here.

**Comment by daniel86** [comment]
> The idea was to get the SAY PR in while Vanja was still here.

Well if you say so. I can only repeat that I am 100% sure that you are wrong

File: owl/SOMA-ELAN.owl [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+        <SOMA:ELANUsageGuideline>walk (to a location, or in a direction)</EASE:ELANUsageGuideline>
+    </rdf:Description>
+
+    <rdf:Description rdf:about="http://www.ease-crc.org/ont/SOMA.owl#BodyMovement">
+        <SOMA:ELANName>move-body-general</EASE:ELANName>
+        <SOMA:ELANUsageGuideline>The Agent moves its body in some way. Use more specific labels where appropriate.</EASE:ELANUsageGuideline>
+    </rdf:Description>
+
+    <rdf:Description rdf:about="http://www.ease-crc.org/ont/SOMA.owl#HeadMovement">
+        <SOMA:ELANName>move-head</EASE:ELANName>
```
Comment (mpomarlan) [inline owl/SOMA-ELAN.owl:64]: Closing tag and opening tags don't match.

**Pull request merged**